components/TextControls/examples/MDCTextControlTypicalUseExample.m

```diff
@@ -0,0 +1,204 @@
+// Copyright 2019-present the Material Components for iOS authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <UIKit/UIKit.h>
+
+#import "MaterialButtons.h"
+#import "MaterialContainerScheme.h"
+#import "MaterialTextControls+Theming.h"
+#import "MaterialTextControls.h"
+
+#import "MDCTextControlTypicalUseExample.h"
+#import "MDCTraitEnvironmentChangeDelegate.h"
+
+@interface MDCTextControlTypicalUseExample ()
+
+/**
+All the content size categories that this view controller supports.
+ */
+@property(nonatomic, strong) NSArray *contentSizeCategories;
+@end
+
+@implementation MDCTextControlTypicalUseExample
```

Comment:
This example doesn't feel like a typical use example anymore because it's not showing typical usage of code we'd expect clients to use. Perhaps something more to the effect of "Demo" or "Configurator"?

Reply:
That's a very good point. This is definitely not a typical use.